src/ctia/task/migration/store.clj

```diff
@@ -634,16 +675,21 @@ when confirm? is true, it stores this state and creates the target indices."
   (update-migration-store migration-id
                           store-key
                           {:completed (time/internal-now)}
-                          es-conn))
-
-(defn setup!
-  "init properties, start CTIA and its store service"
-  []
+                          es-conn
+                          services))
+
+(s/defn setup!
+  "setup store service"
+  [services :- MigrationStoreServices]
+  ;; TODO will be rolled into trapperkeeper bootstrap
+  ;; START global services setup
```

Comment:
So the migration will only use the configuration services, and should now avoid triggering `init--store-services!` that updates source store. This is handy when starting CTIA but should be avoided (and could fail) for a migration.
https://github.com/threatgrid/ctia/pull/968/files#diff-0e5ad98d51479717941d6ba505d26839L640

Reply:
Ok, I'm hesitant to touch this code in this PR, so I'd prefer for one of us to resolve a merge conflict. If you merge #968 I can merge this PR back on master.

Reply:
Deleted after merge.